src/controllers/users.js

```diff
@@ -11,3 +12,34 @@ exports.getUser = async (req, res, next) => {
   // This is an example of a successful response
   res.json({ success: true, data: user }); */
 };
+exports.updateProduct = async (req, res) => {
```

Comment:
You should create a new file `products.js` inside `routes` and `controllers` folder, and add those endpoints from there.

Reply:
done you can check it. I was confused about routes and controllers structure now I got the point ı think thank you